tests/e2e/scenarios/aws-lb-controller/run-test.sh

```diff
@@ -33,6 +33,9 @@ ZONES="eu-west-1a,eu-west-1b,eu-west-1c"
 
 kops-up
 
+export KUBECONFIG=$(mktemp -t kops.XXXXXXXXX)
```

Comment:
https://www.shellcheck.net/wiki/SC2155
```suggestion
KUBECONFIG=$(mktemp -t kops.XXXXXXXXX)
export KUBECONFIG
```

Reply:
Do we need to export?

Reply:
I believe so because the ginkgo process is what is loading the client-go config and reading from the KUBECONFIG env var, not just a shell variable.